.github/workflows/klocwork-pull-request.yml

```diff
@@ -102,17 +102,24 @@ jobs:
           if-no-files-found: error
       
       - name: get child issue ids
-        run: awk -F ';' '{print $1}' /home/klocwork/child_issues.txt |sort > /home/klocwork/child_issues_ids.txt
+        run: awk -F ';' '{print $1}' /home/klocwork/child_issues.txt | sort > /home/klocwork/child_issues_ids.txt
       
       - name: fixed issues
         run: |
           FIXED_ISSUE_ID=$(comm -13 /home/klocwork/child_issues_ids.txt /home/klocwork/parent_issues_ids.txt | sed -z 's/\n/,/g;s/,$/\n/')
```

Comment:
Not entirely sure if this applies here, but there is a nice shortcut to concatenate lines in this manner:
```
paste -sd, <<'EOF'
a
b
c
d
EOF
```
```
a,b,c,d
```

Reply:
Thanks! Its much readable and simpler.